ui/page/components/mixer_layout.go

```diff
@@ -0,0 +1,140 @@
+package components
+
+import (
+	"gioui.org/font"
+	"gioui.org/layout"
+
+	"github.com/crypto-power/cryptopower/ui/cryptomaterial"
+	"github.com/crypto-power/cryptopower/ui/load"
+	"github.com/crypto-power/cryptopower/ui/values"
+)
+
+type MixerComponent struct {
+	*load.Load
+
+	WalletName,
+	UnmixedBalance string
+
+	Width,
+	Height int
+
+	ForwardButton,
+	InfoButton cryptomaterial.IconButton
+}
+
+func (mc MixerComponent) MixerLayout(gtx C) D {
+	r := 8
+	return cryptomaterial.LinearLayout{
+		Width:       mc.Width,
+		Height:      mc.Height,
+		Orientation: layout.Vertical,
+		Padding:     layout.UniformInset(values.MarginPadding15),
+		Background:  mc.Theme.Color.Surface,
+		Border: cryptomaterial.Border{
+			Radius: cryptomaterial.CornerRadius{
+				TopLeft:     r,
+				TopRight:    r,
+				BottomRight: r,
+				BottomLeft:  r,
+			},
+		},
+	}.Layout(gtx,
+		layout.Rigid(mc.topMixerLayout),
+		layout.Rigid(mc.middleMixerLayout),
+		layout.Rigid(mc.bottomMixerLayout),
+	)
+}
+
+func (mc MixerComponent) topMixerLayout(gtx C) D {
+	return layout.Flex{
+		Axis:      layout.Horizontal,
```

Comment:
remove this line because default Axis is Horizontal

Reply:
Yeah.. i left is because it makes the code readable for first time gio users, and would make reading and managing the codes easier.